yarn/default.nix

```diff
@@ -1,10 +1,19 @@
 { callPackage }:
 
 rec {
-  yarn = callPackage ./yarn.nix {};
-  yarn-patched = yarn.override {
-    applyBuiltinPatches = true;
+  yarn-unwrapped = callPackage ./yarn.nix {};
+
+  wrapYarnWith = { yarn, ... } @ extraArgs: callPackage ./wrapper.nix ({
```

Comment:
This is mostly copied blindy from upstream. I don't think it's wrong, and it's clear what it does, but maybe this is simpler?
```nix
wrapYarnWith = { yarn, ... } @ args: callPackage ./wrapper.nix args
```
```